besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1666,6 +1674,47 @@ private GenesisConfigOptions readGenesisConfigOptions() {
     }
   }
 
+  /**
+   * generateEphemeryGenesisFile checks if a genesis update is available
+   * and updates the ephemery.json file
+   */
+  private void generateEphemeryGenesisFile() {
```

Comment:
this could prob be extracted into a standalone class and add a few more tests around it?

Reply:
> this could prob be extracted into a standalone class and add a few more tests around it?

I will create a standalone class and add more tests to it. Thanks for checking!